matrix-vis.js

```diff
@@ -105,11 +105,24 @@ function initGraph() {
     network = new vis.Network(container, data, options);
     // add event listeners
     network.on('select', function(params) {
-        var text = "";
+        var Type = " ";
+        var Sender = " ";
+        var m = " ";
         if (params.nodes.length === 1) {
-            text = JSON.stringify(nodes.get(params.nodes[0]).blob, undefined, 2);
+            Type=JSON.stringify(nodes.get(params.nodes[0]).blob.type, undefined, 2);
+            Sender=JSON.stringify(nodes.get(params.nodes[0]).blob.sender, undefined, 2);
```

Comment:
You do not need to `JSON.stringify` this. The JSON object is called `blob` and you're now grabbing the `string` under the keys `type` and `sender`. They are already strings.